src/types.rs

```diff
@@ -507,6 +507,16 @@ pub struct NewAccount<'a> {
     pub only_return_existing: bool,
 }
 
+/// This is a special value for `NewAccount` that is supplied
+/// when loading an account from a private key.
+/// According to rfc8555 7.3.1 this field needs to be supplied,
+/// but MUST be ignored by the ACME server.
```

Comment:
I think we can describe this a bit clearer:

```suggestion
/// A `NewAccount`  endpoint request payload for fetching an existing account.
///
/// `only_return_existing` is set to true to avoid creating a new account if the
/// existing account is not known to the ACME server for some reason.
```